widget/titter.jsx

```diff
@@ -209,6 +209,8 @@ var React = require('react');
         createTwitterWidget: function (commentId) {
             window.onTimelineLoad = function () {
                 var $twitter = document.getElementsByClassName('twitter-timeline-rendered')[0];
+
+                //NOTE: autoSizeTimeline call multiple times after each re-render CreateTitter component
                 function autoSizeTimeline() {
```

Comment:
@rageofaxe Yes it's a dirty hack but it needed now. It scans for twitter timeling height changes(inside the iframe) and changes height of underlying timeline to eliminate inner scrollbar.
Changes you proposed would break functionality, so double scrollbar will appear when adding more tweets. You need either leave that intact, or find a way to trigger event, when  document.getElementsByClassName('twitter-timeline-rendered') height is changed.